client/state/action-types.js

```diff
@@ -50,6 +50,7 @@ export const RECEIPT_FETCH_FAILED = 'RECEIPT_FETCH_FAILED';
 export const REMOVE_NOTICE = 'REMOVE_NOTICE';
 export const SELECTED_SITE_SET = 'SELECTED_SITE_SET';
 export const SERIALIZE = 'SERIALIZE';
+export const SERVER_DESERIALIZE = 'SERVER_DESERIALIZE';
```

Comment:
I was going to say that `BOOTSTRAP` may give more clue as to what this is for, but I checked back, and that term is not used at all in the server-rendering section of the [Redux docs](http://redux.js.org/docs/recipes/ServerRendering.html), so I think it is fine as is.